src/components/Ship.vue

```diff
@@ -1,25 +1,41 @@
 <template>
-  <article class="swapi-itemSheet_Container">
-    <template v-if="hasData">
-      <h1>I'm a SW {{ singularizeTitle }}!!</h1>
-      <h2>My name is {{ name }}</h2>
-      <h2>I'm a <em>{{ model }}</em> model</h2>
-      <h2>I belong to the <em>{{ starship_class }}</em> class</h2>
-      <films-sub-list
-        :films="films">
-      </films-sub-list>
-      <characters-sub-list
-        v-if="pilots.length"
-        :items="pilots"
-        :itemsName="pilotsTitle">
-      </characters-sub-list>
-    </template>
-    <template v-else>
-      <item-sheet-error-message
-        :sectionTitle="sectionTitle">
-      </item-sheet-error-message>
-    </template>
-  </article>
+<article class="swapi-itemSheet_Container">
+  <template v-if="hasData">
+    <h1>
+      I'm a SW {{ singularizeTitle }}!!
+    </h1>
+    <h2>
+      My name is {{ name }}
+    </h2>
+    <h2>
+      I'm a
+      <em>
+        {{ model }}
+      </em>
+      &nbsp;model
+    </h2>
+    <h2>
```

Comment:
If you are writing non complex text dont over ident, so it's right to write. Is more readable because it's unnatural to nest text.

`<h2>I'm a <em> {{ model }} </em> model </h2>`

Also you can optimize it with a modelName attribute in your data function and use string interpolation

modelName: I'm a <em> {{ model }} </em> model (add ` at the beginning and the end of phrase for string interpolation)

So you can write:

`<h2 v-html="modelName"></h2>`